openroad_interface/README.md

```diff
@@ -0,0 +1,17 @@
+# Openroad Interface
+
+Allows codesign to utilize OpenROAD for resistance, capacitance, and length calculations. The main function takes in two parameters, the graph file name and tcl file directory, and will return a pandas DataFrame containing all information about openroad rcl and estimated rcl.
+
+## Installation
+1. Build OpenROAD using the documentation that they provide: https://openroad.readthedocs.io/en/latest/user/Build.html
+2. Once OpenROAD is built, the command "openroad", which activates the tcl shell, should be able to run everywhere. If this is not the case, just ensure that this command is able to run in the OpenROAD test directory by exporting the route. To do this, go to the OpenROAD test directory (OpenROAD/test), then run:
+```
+export PATH=./../build/src:$PATH
+```
```

Comment:
Change this to be the absolute path to openroad 
you can frame in terms of <path_to_your_codesign_root>/openroad_interface/OpenROAD/build/src
or whatever it should be.